src/operations/rgb.rs

```diff
@@ -7,7 +7,7 @@ mod import_asset;
 mod issue_asset;
 mod receive_tokens;
 mod send_tokens;
-pub mod shared;
+mod shared;
```

Comment:
Be sure to leave this like it is in `development`:
https://github.com/diba-io/bitmask-core/blob/a23e8efe3031a8cf6fad3fd507f4942a0d95a97e/src/operations/rgb.rs#L10